api/envoy/extensions/filters/network/thrift_proxy/v3/thrift_proxy.proto

```diff
@@ -93,6 +106,9 @@ message ThriftProxy {
 
   // Optional maximum requests for a single downstream connection. If not specified, there is no limit.
   google.protobuf.UInt32Value max_requests_per_connection = 7;
+
+  // Use xDS to fetch the route configuration. It is invalid to define both route_config and trds.
+  Trds trds = 8 [(udpa.annotations.field_migrate).oneof_promotion = "route_specifier"];
```

Comment:
nit: let's move this up next to the `route_config` field.

Reply:
Done